internal/pkg/vm/internal/vmcontext/vmcontext.go

```diff
@@ -424,18 +449,23 @@ func (vm *VM) applyMessage(msg *types.UnsignedMessage, onChainMsgSize int, rnd c
 	// 2. build invocation context
 	// 3. process the msg
 
+	topLevel := topLevelContext{
+		originatorStableAddress: senderState.Address,
+		originatorCallSeq:       0,
```

Comment:
`originatorCallSeq: fromActor.CallSeqNum`?

not sure if you want it before or after the `fromActor.IncrementSeqNum()`

Reply:
Oops, thanks. Used `msg.CallSeqNum`